src/submission/migration/migration-manager.ts

```diff
@@ -236,6 +237,8 @@ export namespace MigrationManager {
 
     const analysis = await dictionaryManagerInstance().analyzeChanges(currVersion, toVersion);
 
+    const diff = await dictionaryManagerInstance().fetchDiff(currVersion, toVersion);
```

Comment:
I think what we should do is update the analyze changes method instead of fetching diff again
it's already fetched in the analysis step